main.go

```diff
@@ -30,8 +34,87 @@ func main() {
 			return
 		}
 
-		if m.Content == "hello" || m.Content == "Hello" {
-			s.ChannelMessageSend(m.ChannelID, "world!")
+		args := strings.Split(m.Content, " ")
+
+		if args[0] != prefix {
+			return
+		}
+
+		if args[1] == "help" {
+			commands := []string{
+				"**help**: Displays all the current integrated commands",
+				"**activity**: Gives a random activity to do in Old School Runescape",
+				"**boss**: Gives a random boss to do in Old School Runescape",
+				"**test**: Replies with initiated (only meant as a test)",
```

Comment:
Perhaps now is a good time to learn about go tests... We can keep this in for now...

Reply:
If you don't want to do that here - very likely don't need to yet, just add a TODO somewhere near `test` so we can add it later.
```suggestion
				"**test**: Replies with initiated (only meant as a test)",
				// TODO: Add go tests later.
```